stdlib/std.ncl

```diff
@@ -2284,7 +2559,8 @@
       = from,
 
     from_enum
-      | std.enum.Tag -> String
+      | forall a. [|; a |] -> String
```

Comment:
Was this meant to be the type rather than a contract?

Reply:
It looks like it, but it's not. Ideally that would be the case but the implementation is using `from` (which is just a wrapper around `%to_str%`), which has a type that is too generic. However, it has the same effect as being a type annotation for consumers of the function, because when several annotations are present, the first one is selected as the "official" type.

You can see it as a simpler/more efficient way of writing something like:

```nickel
from_enum
      : forall a. [|; a |] -> String
      | std.enum.Tag -> Dyn
      | doc m%"
        Converts an enum variant to its string representation.

        # Examples

        ```nickel
        std.string.from_enum 'MyEnum
          => "MyEnum"
        ```
      "%
      = from | forall b. [|; b|] -> String,
```

But maybe it's confusing, when you read the interface of the function...